### DISCUSSION TITLE
A way to edit existing label?

### DISCUSSION THREAD

**Discussion by gked** [TriliumNext/Trilium#2288] [Q&A] [answered]
Hey everyone,

Is there any way to edit existing labels? I can't, for the life of me figure out how to do it.

Thanks,
gked

**Comment by abitofevrything** [comment, marked as answer]
Open the owned attributes bar on a note, find the attribute you want to edit and then change the value to whatever you want. Labels look like `#name=value`, possibly with quotes around `value` if it contains spaces.

**Comment by zadam** [comment]
Besides what @abitofevrything wrote it's also possible to click on the label which will bring up this dialog:
![image](https://user-images.githubusercontent.com/617641/139244497-1f62ae62-2449-4a39-8ac6-cd37f547c7dc.png)
